calendar-app/Components/API/Invites/InviteToGroup.ts

```diff
@@ -0,0 +1,23 @@
+import backendURL from "../BeUrl";
+
+export async function inviteToGroup({ user_id, group_id }) {
```

Comment:
It would be a good practice to create an interface, so the api request would be more developer-safe